packages/schema/src/cli/actions/repl.ts

```
@@ -63,7 +70,11 @@ export async function repl(projectPath: string, options: { prismaClient?: string
                 let r: any = undefined;
                 let isPrismaCall = false;
 
-                if (cmd.includes('await ')) {
+                if (/^\s*user\s*=[^=]/.test(cmd)) {
+                    // assigning to user variable, reset auth
+                    eval(cmd);
```

Biome

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)